src/appBootstrapper.tsx

```diff
@@ -235,6 +237,26 @@ if (__DEBUG__ && module.hot) {
     module.hot.accept('./routes', () => render());
 }
 
+async function loadCustomJs() {
+    const customJsFiles = getServerConfig().custom_js_files;
+    if (!customJsFiles) {
+        return Promise.resolve();
+    }
+    getServerConfig().customJsFunctions = {};
+    return Object.values(customJsFiles).map(
+        (customJsFile: CustomJsFile) =>
+            new Promise((resolve, reject) => {
```

Comment:
i'm not 100% sure but you are returning an array of promises, not a promise.  So my guess is that ```await loadCustomjs``` down below doesn't actually await the promises.  i think you want you want is to return Promise.all() on this map.

Reply:
Fixed in 96026be